src/main/java/com/my/relink/controller/review/dto/request/ReviewReqDto.java

```diff
@@ -0,0 +1,23 @@
+package com.my.relink.controller.review.dto.request;
+
+import com.my.relink.domain.review.TradeReview;
+import jakarta.validation.constraints.Digits;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.math.BigDecimal;
+
+@Getter
+@AllArgsConstructor
+public class ReviewReqDto {
+
+    @Digits(integer = 1, fraction = 2)
+    @NotNull(message = "별점은 소수점 1자리까지 허용됩니다.")
+    private BigDecimal star;
```

Comment:
fraction = 2 는 소수점 2자리까지 허용하는 것으로 알고있습니다. 
그리고 NotNull 메세지 내용은 NotNull에 해당하는 것이 아니기 때문에 해당 메세지를 Digits에 옮겨주시는 게 좋아보입니다.
```suggestion
    @Digits(integer = 1, fraction = 1, message = "별점은 소수점 1자리까지 허용됩니다.")
    @NotNull(message = "별점을 입력해주세요.")
    private BigDecimal star;
```

Reply:
넵 수정하겠습니다!